packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -135,8 +135,8 @@ class MenuContent extends React.Component {
 
   validateFocus(node) {
     if (this.needsFocus && node) {
-      node.focus();
-      this.needsFocus = document.activeElement !== node;
```

Comment:
This shifts focus to the first menu item in the menu when opened, (this first interactable element in the menu).